typewritten.zsh

```diff
@@ -26,10 +26,15 @@ if [ ! -z "$TYPEWRITTEN_SYMBOL" ]; then
   tw_prompt_symbol="$TYPEWRITTEN_SYMBOL"
 fi;
 
-local tw_prompt_color="%(?,%F{$tw_colors[symbol]},%F{$tw_colors[error_code]})"
+local tw_prompt_symbol_color="%F{$tw_colors[symbol]}"
+if [[ $(id -u) -eq 0 ]]; then
+  tw_prompt_symbol_color="%F{$tw_colors[symbol_root]}"
+fi
+
+local tw_prompt_color="%(?,$tw_prompt_symbol_color,%F{$tw_colors[error_code]})"
 local tw_return_code="%(?,,%F{$tw_colors[symbol_error]}%? )"
```

Comment:
Just noticed I made a mistake with the color variables here, `tw_prompt_color` uses `tw_colors[error_code]` and  `tw_return_code` uses `tw_colors[symbol_error]`. It should be the opposite.